packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -29,25 +29,18 @@ import {
 export const defaultLayouts = {
 	[ LAYOUT_TABLE ]: {
 		layout: {
-			primaryField: 'title',
 			styles: {
-				title: {
+				__primary: {
 					maxWidth: 300,
 				},
 			},
 		},
 	},
 	[ LAYOUT_GRID ]: {
-		layout: {
-			mediaField: 'featured_media',
-			primaryField: 'title',
-		},
+		layout: {},
 	},
 	[ LAYOUT_LIST ]: {
-		layout: {
-			primaryField: 'title',
-			mediaField: 'featured_media',
-		},
+		layout: {},
```

Comment:
The list layout doesn't support `layout`. Why does it need to pass an empty object?

Reply:
I presume we've done this for grid as well to reset the incoming values we could potentially have when switching from table. Can we reset the layout if the new view doesn't have the property instead?

Reply:
It probably doesn't need it.